packages/account/src/Components/personal-details/personal-details.jsx

```diff
@@ -202,7 +206,7 @@ const PersonalDetails = ({
                                 <FormSubmitButton
                                     cancel_label={localize('Previous')}
                                     has_cancel
-                                    is_disabled={isSubmitDisabled(errors)}
+                                    is_disabled={should_scroll_to_error_field ? false : isSubmitDisabled(errors)}
```

Comment:
Button will only be disabled while submitting the form, so we can remove these flags and just use `isSubmitting` from `Formik`

Reply:
Alright, but for falsy should_scroll_to_error_field let’s keep current implementation

Reply:
This scroll behaviour will be default for the forms, we won't even need this flag

Reply:
personal-details component is supposed to be reused, but currently we have the requirements to use scroll only for real account creation